assets/js/base/context/providers/cart-checkout/payment-methods/constants.ts

```diff
@@ -8,6 +8,7 @@ import type {
 
 export enum STATUS {
 	PRISTINE = 'pristine',
+	IDLE = 'idle',
```

Comment:
I suggest documenting these statuses for a better understanding in the future. Comments above each of them should do it.